src/components/ActionsPanel/components/ActionItem.tsx

```diff
@@ -51,7 +51,7 @@ const IconContainer = styled.div`
   height: 28px;
 `;
 
-const TitleContainer = styled.div`
+export const TitleContainer = styled.div`
```

Comment:
Why the export?

Reply:
It was used for something I removed, fixing it